examples/spark/src/test/java/edu/snu/onyx/examples/spark/SparkITCase.java

```diff
@@ -21,34 +21,45 @@
 import org.junit.Before;
 import org.junit.Test;
 import org.junit.runner.RunWith;
+import org.powermock.core.classloader.annotations.PowerMockIgnore;
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
 
 /**
- * Test SparkPi program with JobLauncher.
+ * Test Spark programs with JobLauncher.
  */
 @RunWith(PowerMockRunner.class)
 @PrepareForTest(JobLauncher.class)
-public final class SparkPiITCase {
+@PowerMockIgnore("javax.management.*")
+public final class SparkITCase {
   private static final int TIMEOUT = 60000;
-  private static final String numParallelism = "3";
-
-  private static ArgBuilder builder = new ArgBuilder()
-      .addJobId(SparkPiITCase.class.getSimpleName())
-      .addUserMain(JavaSparkPi.class.getCanonicalName())
-      .addUserArgs(numParallelism);
+  private static ArgBuilder builder = new ArgBuilder();
 
   @Before
-  public void setUp() throws Exception {
-    builder = new ArgBuilder()
-        .addUserMain(JavaSparkPi.class.getCanonicalName())
-        .addUserArgs(numParallelism);
+  public void setUp() {
+    builder = new ArgBuilder();
   }
 
   @Test(timeout = TIMEOUT)
-  public void test() throws Exception {
+  public void testSparkWordCount() throws Exception {
```

Comment:
Why don't we test WC with multi parallelism also?

Reply:
The example program wasn't written like that, but we could add that with the `repartition` method.